device-api/src/device.rs

```diff
@@ -291,61 +291,64 @@ pub struct DeviceInfo {
     device_index: u8,
     dev_root: PathBuf,
     sys_root: PathBuf,
-    meta: DeviceMetadata,
+    arch: Arch,
+    meta: Mutex<HashMap<&'static str, String>>,
```

Comment:
`meta` doesn't seem to update its value after creation in `new`, does it need to be a `Mutex`?

Reply:
Fix at [b1e2ce5](https://github.com/furiosa-ai/device-api/pull/83/commits/b1e2ce55cbd06bae1c92a0604310a11664018b8a)